Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/Visuals/RotationHandles.cs

```diff
@@ -49,7 +49,7 @@ protected override Quaternion GetRotationRealignment(int handleIndex)
             }
             else // y axis 
             {
-                return Quaternion.FromToRotation(Vector3.forward, Vector3.up);
+                return Quaternion.identity;
```

Comment:
This seems like a different behavior (no rotation vs rotation 90 degrees about x) -- is that intentional?

Reply:
yes! this was a mistake carried over from translation handles. this always used to be identity. when i did the translation handles integration this sneaked in as a mistake and flattened rotation handles had the wrong rotation applied